src/App.js

```diff
@@ -1,14 +1,25 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import './App.css';
-import Calculator from './components/calculator';
-import Quote from './components/displayQuote';
+import Calculator from 'src/components/Calculator';
```

Comment:
Please fix the import statement for the `Calculator` component. The file name is `calculator` with lowercase c. You are also importing it from the wrong directory.